doc/ledmon.conf.pod

```diff
@@ -42,8 +42,8 @@ See also the examples section.
 
 =head2 List of configurable options:
 
-B<BLACKLIST> - Ledmon will exclude scanning controllers listed on blacklist.
```

Comment:
If you look at the commit, it is backwards compatible.

Reply:
OK, thanks I didn't notice.
Anyway, I would like to add it in separate pull request because I'm tending to squash this pr.
Later, I may forgot about the change but I must be described in release notes.

Reply:
Resolved with https://github.com/intel/ledmon/pull/118